### DISCUSSION TITLE
in the future, Application only filter for categories

### DISCUSSION THREAD

**Discussion by Quackdoc** [EmiyaSyahriel/CrossLauncher#65] [General]
I know in the past cross launcher had support for showing stuff inside of the categories like music and videos. And while this is neat, I would like it if we could only show apps even if only via config. There are good open source controller friendly music players, Image viewers and video players. and it would be nice to show just the apps themselves.

**Comment by EmiyaSyahriel** [comment]
Sure, The latest action build is actually doing just this since the media entry listing code hasn't yet fully implemented, You can just change the targeted media app's Category in it's app info page and the category icon will just show up with only app in it.

And I'm planning to make the media entry listing can be switched on and off without removing the category icon in the menu, So I think there is no need to change any code here yet, But I'll transfer this to issue in case I'm not remember to implement this feature.

**Comment by Quackdoc** [comment]
Thanks for the consideration, my only remaining concerns now are the issues with phones not launching into leanback mode. thing which I plan on looking into and auto sorting applications which is something I think I can look into myself too. Thanks a bunch for willingness to support
